core/templates/pages/oppia-root/routing/access-validation-backend-api.service.ts

```diff
@@ -64,10 +64,6 @@ export class AccessValidationBackendApiService {
     return this.http.get<void>(url).toPromise();
   }
 
-  accountDeletionIsEnabled(): Promise<void> {
```

Comment:
Hm, shouldn't you get rid of the `ACCOUNT_DELETION_IS_ENABLED` constant above as well?

Let's clean up everything that is orphaned as a result of removing whatever is being removed in this PR, so that we don't have any dead code. Thanks!

Reply:
Yep, I forgot to remove it after trying to fix a failing test.
Done.